tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -266,7 +266,7 @@ The exact location of the Python library depends on your system, but is usually
 /usr/local/lib/python2.7/site-packages/tensorflow
 ```
 
-You can find out the directory with the following command:
+You can find out the directory with the following command (make sure to use the Python you installed TensorFlow to, for example, use ```python3``` instead of ```python``` if you installed for Python 3):
```

Comment:
Can you use single backticks instead of triple ones for the inline code? Like this: `python`?